docs/docs/tutorials/creating_main.go.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+title: Creating main.go
+parent: Tutorials
+nav_order: 4
+---
+
+## Creating main.go
+
+For `main.go` file of hello world we can refer to the boilerplate repository [grpc-gateway-boilerplate Template](https://github.com/johanbrandhorst/grpc-gateway-boilerplate).
```

Comment:
OK we shouldn't be doing this here. We can refer to the boilerplate repo at the end of the tutorial for a "read more, complete example" link. Here we should include everything necessary to run a basic gRPC server, in a code example. You can copy most of `main.go` from the boilerplate repo and just include the _essentials_, remove everything else.